selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -349,45 +263,20 @@ public void checkWorkspaceActions() throws Exception {
 
     workspaces.waitPageLoading();
     workspaces.waitVisibleWorkspacesCount(getWorkspacesCount());
-
-    Workspaces.WorkspaceListItem newestCreatedWorkspaceItem =
-        workspaces.getWorkspacesListItemByWorkspaceName(
-            workspaces.getVisibleWorkspaces(), NEWEST_CREATED_WORKSPACE_NAME);
-
-    assertTrue(newestCreatedWorkspaceItem.equals(expectedNewestWorkspaceItem));
   }
 
-  @Test
-  public void deleteWorkspacesByCheckboxes() throws Exception {
+  @Test(priority = 1)
+  public void deleteWorkspacesByCheckboxes() {
     workspaces.waitPageLoading();
 
-    workspaces.selectWorkspaceByCheckbox(workspaceToDelete.getName());
+    workspaces.selectWorkspaceByCheckbox(NEWEST_CREATED_WORKSPACE_NAME);
     workspaces.clickOnDeleteWorkspacesBtn();
     workspaces.clickOnDeleteButtonInDialogWindow();
 
-    workspaces.waitWorkspaceIsNotPresent(workspaceToDelete.getName());
-  }
-
-  private void checkExpectedBlankWorkspaceDisplaying() {
-    List<Workspaces.WorkspaceListItem> items = workspaces.getVisibleWorkspaces();
-
-    Workspaces.WorkspaceListItem currentDisplayingBlankItem =
-        workspaces.getWorkspacesListItemByWorkspaceName(
-            items, expectedBlankItem.getWorkspaceName());
-
-    assertEquals(currentDisplayingBlankItem, expectedBlankItem);
-  }
-
-  private void checkExpectedJavaWorkspaceDisplaying() {
-    List<Workspaces.WorkspaceListItem> items = workspaces.getVisibleWorkspaces();
-
-    Workspaces.WorkspaceListItem currentDisplayingJavaItem =
-        workspaces.getWorkspacesListItemByWorkspaceName(items, expectedJavaItem.getWorkspaceName());
-
-    assertEquals(currentDisplayingJavaItem, expectedJavaItem);
+    workspaces.waitWorkspaceIsNotPresent(NEWEST_CREATED_WORKSPACE_NAME);
   }
 
   private int getWorkspacesCount() throws Exception {
```

Comment:
It duplicates `TestWorkspaceServiceClient::getWorkspacesCount()` and so can be removed.